src/lib/renderComponent.tsx

```diff
@@ -118,6 +119,21 @@ const renderWithFocusZone = (render, focusZoneDefinition, config, focusZoneRef):
   return render(config)
 }
 
+const getProviderNotUsedWarningElement = (displayName: string) => {
+  console.error(`Incorrect usage. Your component needs to be wrapped using the Provider component:
```

Comment:
We should probably also implement a cache check so we don't spam the console for every component on the page.

Reply:
done